src/opnsense/www/js/opnsense_theme.js

```diff
@@ -30,176 +30,180 @@
+          transition_duration(350);
         }
-    }
-
-    if (toggle_sidebar_loaded) {
-        var toggle_btn = $(".toggle-sidebar");
-        /* navigation toggle */
-        toggle_btn.click(function () {
-            opnsense_sidebar_toggle(true);
-            $(this).blur();
-        });
-
-        /* sidebar mouseenter */
-        navigation.mouseenter(function () {
-            if (navigation.hasClass("col-sidebar-left")) {
-                transition_duration(0);
-                layer1_a.on(events);
-                layer2_a.on(events);
-                layer1_div.on(events2);
-                layer2_div.on(events2);
-            }
-        });
-
-        /* sidebar mouseleave */
-        $("#mainmenu").mouseleave(function () {
-            if ($("#navigation").hasClass("col-sidebar-left")) {
-                layer1_a.attr("aria-expanded", "false").next("div").removeClass("in");
-                layer2_a.attr("aria-expanded", "false").next("div").removeClass("in");
-                layer1_div.removeAttr("style");
-                layer2_div.removeAttr("style");
-            }
-        });
-
-        /* on resize - toggle sidebar / main navigation */
-        $(window).on('resize', function () {
-            var win = $(this);
-            winheight = win.height();
-            if ((win.height() < 675 || win.width() < 760) && navigation.not("col-sidebar-hidden")) {
-                navigation.addClass("col-sidebar-hidden");
-                mouse_events_off();
-                if (navigation.hasClass("col-sidebar-left")) {
-                    opnsense_sidebar_toggle(false);
-                    mouse_events_off();
-                    transition_duration(350);
-                }
-            } else if ((win.height() >= 675 && win.width() >= 760) && navigation.hasClass("col-sidebar-hidden")) {
-                $("#navigation").removeClass("col-sidebar-hidden");
-                transition_duration(0);
-                if (window.sessionStorage && sessionStorage.getItem('toggle_sidebar_preset') == 1) {
-                    opnsense_sidebar_toggle(false);
-                }
-            }
-        });
-
-        /* only show toggle button when style is loaded */
-        toggle_btn.show();
-
-        /* auto-collapse if previously requested */
+      } else if ((win.height() >= 675 && win.width() >= 760) && navigation.hasClass('col-sidebar-hidden')) {
+        navigation.removeClass('col-sidebar-hidden');
+        transition_duration(0);
         if (window.sessionStorage && sessionStorage.getItem('toggle_sidebar_preset') == 1) {
-            opnsense_sidebar_toggle(false);
+          opnsense_sidebar_toggle(false);
         }
+      }
+    });
+    
+    /* only show toggle button when style is loaded */
+    toggle_btn.show();
+    
+    /* auto-collapse if previously requested */
+    if (window.sessionStorage && sessionStorage.getItem('toggle_sidebar_preset') == 1) {
+      opnsense_sidebar_toggle(false);
     }
-});
+  }
+});
```

Comment:
This indicates the final newline is missing from the file. Also good to teach the IDE to do this. :)